### PR TITLE
adjusted speed to message length

### PR DESCRIPTION
**Pull request by wbaccinelli**
Fixes: https://github.com/PerfectFit-project/virtual-coach-issues/issues/360 

The messages sending thorough the API happens when the dialogs are externally triggered. The response is sent to the API through the `custom_channels.py`. A delay between one message and the other has been added there.

### CODE REVIEW

File: Rasa_Bot/custom_channels.py [hunk 1]
```diff
@@ -13,6 +14,11 @@
 
 
 NICEDAY_API_URL = os.getenv('NICEDAY_API_ENDPOINT')
+# proportional factor between the number of words in a message and the time to wait before the
+# newt message is delivered
```
Comment (raar1) [inline Rasa_Bot/custom_channels.py:18]:
```suggestion
# next message is delivered
```

File: Rasa_Bot/custom_channels.py [hunk 2]
```diff
@@ -13,6 +14,11 @@
...
+WORDS_PER_SECOND = 5
```
Comment (raar1) [inline Rasa_Bot/custom_channels.py:19]: Haha, ok, so this is the fix for the PF app spamming users with 6000 messages a minute like a deranged stalker?

File: Rasa_Bot/custom_channels.py [hunk 3]
```diff
@@ -13,6 +14,11 @@
...
+# maximum delay between a message and the next one
+MAX_DELAY = 10
```
Comment (raar1) [inline Rasa_Bot/custom_channels.py:21]: Also units of seconds, I assume?

**Review by raar1 — APPROVED** [review]

File: Rasa_Bot/custom_channels.py (hunk 2 above)
Comment (wbaccinelli) [reply Rasa_Bot/custom_channels.py:19]: ahahah yes, now it should be less annoying (if possible).

File: Rasa_Bot/custom_channels.py (hunk 3 above)
Comment (wbaccinelli) [reply Rasa_Bot/custom_channels.py:21]: yep, I'm updating the definition

**Pull request merged**